.idea/codeStyles/Project.xml

```diff
@@ -47,6 +46,9 @@
           <emptyLine />
         </value>
       </option>
+      <option name="ALIGN_MULTILINE_RECORDS" value="false" />
```

Comment:
Do these changes belong in this PR? What are they for?

Reply:
No, they do not belong. They change the IntelliJ settings to prevent auto-alignment, since I feel that the kind of auto-alignment that IntelliJ uses is a waste of horizontal space. They also move the vertical lines in the editor to 80 columns and 100 columns in accordance with our decision to keep comments 80 columns wide and code 100 columns wide.

There are a few things we could do about this:
- Stop tracking the IntelliJ settings so that each programmer can adapt the settings to their personal preferences without losing the changes to the settings whenever they need to pull or switch branches;
- Move these changes into a separate PR, with the understanding that they reinforce certain code style expectations;
- Undo these changes, and wait until we have a Java style guide before applying them.

Reply:
The changes make sense. I vote for moving them to a separate PR, but I'm also fine with sneaking them in -- I mostly just wanted to understand what these changes meant :-)

Reply:
> I vote for moving them to a separate PR

Done!